### PR TITLE
Update/option page

### PR DESCRIPTION
**Pull request by marcelomazza**
Initial commit for the Install page. MVP, will iterate on it. See detailed changelog in commit message:

https://github.com/marcelomazza/Press-This/commit/fea64d66978b0d92d56ddbe337a5b0f13905e294

### CODE REVIEW

File: press-this.php [hunk 1]
```diff
@@ -983,23 +983,82 @@ public function register_options_page() {
...
-				<p class="pressthis"><a onclick="return false;" oncontextmenu="if(window.navigator.userAgent.indexOf('WebKit')!=-1||window.navigator.userAgent.indexOf('MSIE')!=-1){jQuery('.pressthis-code').show().find('textarea').focus().select();return false;}" href="<?php echo htmlspecialchars( get_shortcut_link() ); ?>"><span><?php _e('Press This') ?></span></a></p>
-				<div class="pressthis-code" style="display:none;">
-					<p class="description"><?php _e('If your bookmarks toolbar is hidden: copy the code below, open your Bookmarks manager, create new bookmark, type Press This into the name field and paste the code into the URL field.') ?></p>
-					<p><textarea rows="5" cols="120" readonly="readonly"><?php echo htmlspecialchars( get_shortcut_link() ); ?></textarea></p>
+			</div>
+			<form>
+			<div class="postbox postbox-pt">
+				<h3><?php _e('How to install it?'); ?></h3>
+				<h4><?php _e('Press This Bookmarklet'); ?></h4>
+				<p><?php _e('This method is faster and almost instantaneous. To use it, drag-and-drop the following link to your bookmarks bar:'); ?></p>
```
Comment (MichaelArestad) [inline press-this.php]: "This method is faster and almost instantaneous." > "The bookmarklet allows you to quickly get content from any site."

File: press-this.php [hunk 2]
```diff
@@ -983,23 +983,82 @@ public function register_options_page() {
...
+
+				<div class="postbox-pt-buttons">
+
+					<a class="button button-primary button-pt-bookmarklet" onclick="return false;" href="<?php echo htmlspecialchars( get_shortcut_link() ); ?>"><?php _e('Press This') ?></a>
+					<?php _e('or'); ?>
+					<a onclick="if(window.navigator.userAgent.indexOf('WebKit')!=-1||window.navigator.userAgent.indexOf('MSIE')!=-1){jQuery('.pressthis-code').show().find('textarea').focus().select();return false;}" oncontextmenu="if(window.navigator.userAgent.indexOf('WebKit')!=-1||window.navigator.userAgent.indexOf('MSIE')!=-1){jQuery('.pressthis-code').show().find('textarea').focus().select();return false;}" href="<?php echo htmlspecialchars( get_shortcut_link() ); ?>"><?php _e('Copy Press This Bookmarklet') ?></a>
+
+					<div class="pressthis-code" style="display: none;">
+						<p><?php _e('If your bookmarks toolbar is hidden: copy the code below, open your Bookmarks manager, create new bookmark, type Press This into the name field and paste the code into the URL field.') ?></p>
```
Comment (MichaelArestad) [inline press-this.php]: "If your bookmarks toolbar is hidden" > "If you can't add it to your bookmarks by dragging,"

File: press-this.php [hunk 3]
```diff
@@ -983,23 +983,82 @@ public function register_options_page() {
...
+						<p><textarea rows="5" cols="120" readonly="readonly"><?php echo htmlspecialchars( get_shortcut_link() ); ?></textarea></p>
+					</div>
+
 				</div>
-				<h3>Bookmarklet code</h3>
-				<p><textarea rows="5" cols="120" readonly="readonly"><?php echo htmlspecialchars( get_shortcut_link() ); ?></textarea></p>
-				<h3>Direct link</h3>
-				<p><textarea rows="1" cols="120" readonly="readonly"><?php echo htmlspecialchars( admin_url( 'press-this.php' ) ); ?></textarea></p>
+				<p><?php _e('The downside is that some mobile browsers don\'t let you add bookmarklets. In that case you should try:'); ?></p>
```
Comment (MichaelArestad) [inline press-this.php]: "The downside is that some mobile browsers don\'t let you add bookmarklets. In that case you should try:" 

I would remove this.

File: press-this.php (hunk 1 above)
Comment (MichaelArestad) [inline press-this.php]: And at the end: "Some mobile browsers make it impossible to add Javascript bookmarklets. For those, use the direct link version below."

**Comment by MichaelArestad** [comment]
Can @azaozz or @stephdau do a quick code review and merge?

**Comment by stephdau** [comment]
Tested it, looking very much better!

Only tiny thing I'd like @MichaelArestad and you to think about is if we should have `onclick="return false;"` on the direct link to PT: https://cloudup.com/cU8BFjZVkuh

As a desktop user, I was expecting to be brought there by clicking on the link. I understand it might be to make it easier to copy on mobile, and probably has more value when applied to the bookmarklet JS above it, but maybe we should let the direct link do what it's supposed to.

Not a show stopper. Will merge. :)

**Pull request merged**